scripts/coverage_report.sh

```diff
@@ -96,6 +96,10 @@ echo "Running tests..."
 while read -r line; do
         subdir=$(dirname "$line");
         dirline=$(realpath "$subdir");
+        if [ "${subdir}" == "." ]; then
```

Comment:
Only virtual crate that gets included, just skipping it, rather than letting it fail makes more sense.